controllers/reclaimspacejob_controller_test.go

```diff
@@ -176,3 +176,42 @@ func TestCalculateReclaimedSpace(t *testing.T) {
 		})
 	}
 }
+
+func Test_targetDetails_canNodeReclaimSpace(t *testing.T) {
+	type fields struct {
+		driverName string
+		pvName     string
```

Comment:
pv name and drivername are not required?

Reply:
> pv name and drivername are not required?

optional, added them now to keep it consistent in the tests.